fuels-signers/Cargo.toml

```diff
@@ -11,8 +11,10 @@ description = "Fuel Rust SDK signers."
 [dependencies]
 async-trait = { version = "0.1.50", default-features = false }
 bytes = { version = "1.1.0", features = ["serde"] }
+fuel-core = { version = "0.3.2", default-features = false, features = ["test-helpers"] }
+fuel-gql-client = { version = "0.3.2", default-features = false }
 fuel-tx = "0.5"
-fuel-types = { version = "0.1", default-features = false }
+fuel-types = { version = "0.1", default-features = false, features = ["random"] }
```

Comment:
Typically we only enable the "random" features in the dev-deps section, since they're only used in testing. This makes use of v2 feature resolver to avoid unifying random impls into the production code.

Reply:
Yeah, nice catch. Just updated it.